engine/resources/resources_test.go

```diff
@@ -174,7 +177,24 @@ func FileExpect(p, s string) Step { // path & string
 	}
 }
 
-// FileExpect takes a path and a string to write to that file, and builds a Step
+// FileOwnerExpect takes a path and a uid to expect from that file, and builds
+// a Step that checks that out of them.
+func FileOwnerExpect(p, o string) Step {
+	return &manualStep{
+		action: func() error { return nil },
+		expect: func() error {
+			fileInfo, _ := os.Stat(p)
+			owner := fileInfo.Sys().(*syscall.Stat_t).Uid
```

Comment:
Some of this can panic, so it would probably be safer to do it the "right" way where we check as we go...
Probably not urgent in a test, but a good learning experience anyways and gives us a clean message instead of a panic.

Reply:
I changed the way we get the file attributes, should be better!